pkgs/development/python-modules/torch-tb-profiler/default.nix

```diff
@@ -43,14 +42,16 @@ buildPythonPackage rec {
     "test_tensorboard_end2end"
     "test_tensorboard_with_path_prefix"
     "test_tensorboard_with_symlinks"
+    "test_autograd_api"
+    "test_profiler_api_with_record_shapes_memory_stack"
+    "test_profiler_api_without_record_shapes_memory_stack"
+    "test_profiler_api_without_step"
```

Comment:
what causes these tests to fail?

Reply:
"Tests that attempt to access the filesystem in naughty ways."